src/components/countries/CountriesPage.tsx

```diff
@@ -5,13 +5,15 @@ import AdminContainer from 'components/admin/navigation/AdminContainer'
 
 import HeadingSeparator from './HeadingSeparator'
 import CountryGrid from './grid/CountryGrid'
+import GridAppbar from './grid/GridAppbar'
 
 export default function CountriesPage() {
   const { t } = useTranslation('countries')
 
   return (
     <AdminLayout>
       <AdminContainer title={t('headings.countries')}>
+        <GridAppbar></GridAppbar>
```

Comment:
Self-closing syntax is prefered when the component has no children

```suggestion
        <GridAppbar />
```